src/core/InsightContainer/index.js

```diff
@@ -195,9 +198,11 @@ function InsightContainer({
               {title}
             </TypographyLoader>
             <BlockLoader loading={loading} height={300}>
-              <Box className={classes.highlightContentChild}>
-                {highlightChild}
-              </Box>
+              {variant === 'data' && (
```

Comment:
```suggestion
              {highlightChild && (
```

We've already checked `variant` above